src/kfactory/kcell.py

```diff
@@ -1134,14 +1134,19 @@ def show(
             library_save_options=library_save_options,
         )
 
-    def plot(self, lyrdb: Path | str | None = None) -> None:
+    def plot(
+        self, lyrdb: Path | str | None = None, display_type: str | None = None
```

Comment:
This can only be "image" or widget, right?
```suggestion
        self, lyrdb: Path | str | None = None, display_type: Literal["image", "widget"] | None = None
```